packages/core/src/common/k8s-api/kube-api.ts

```diff
@@ -97,7 +97,7 @@ export interface DerivedKubeApiOptions {
 
 export interface KubeApiQueryParams {
   watch?: boolean | number;
-  resourceVersion?: string;
+  resourceVersion?: string | number;
```

Comment:
All of the references to `resourceVersion` here (https://kubernetes.io/docs/reference/using-api/api-concepts/#resource-versions) only talk about `string`, why do you think it can also be a number?

Reply:
Furthermore, all references to `resourceVersion` in the kubernetes client-go library are for either `string` or `*string`